lib/Runtime/Library/UriHelper.cpp

```diff
@@ -225,7 +225,7 @@ namespace Js
                 uint32 utfLen = ToUTF8(uVal, bUTF8);
                 for( uint32 j = 0; j < utfLen; j++ )
                 {
-#pragma prefast(suppress: 26014, "buffer length was calculated earlier");
+#pragma prefast(disable: 26014, "buffer length was calculated earlier");
```

Comment:
Could you instead define a return range for ToUTF8 to handle L229, and _analysis_assume that outCurrent is low enough that it has the space for the three writes?

Reply:
I tried but it wasn't happy. I think in this case I'd want to clean up the other analysis_assumes to stop confusing it, but I think this fix is OK for now.